sql/slave.cc

```diff
@@ -5050,6 +5050,7 @@ pthread_handler_t handle_slave_sql(void *arg)
 
 #ifdef WITH_WSREP
   thd->wsrep_exec_mode= LOCAL_STATE;
+  wsrep_thd_set_query_state(thd, QUERY_EXEC);
```

Comment:
This seems fine but can you give some pointer how to handle this on 10.4 or is it so that this issue does not repeat on 10.4 ?